### PR TITLE
Ag/devise test branch

### PR DESCRIPTION
**Pull request by agallant121**
Add user model to db along with user model spec and user.rb files

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -28,6 +28,8 @@ gem 'jbuilder', '~> 2.7'
 # Reduces boot times through caching; required in config/boot.rb
 gem 'bootsnap', '>= 1.4.2', require: false
 gem 'rspec-rails'
+gem 'devise'
```
Comment (TravisBorgsmiller) [inline Gemfile:31]: This gem is really useful!

**Comment by caachz** [comment]
looks good. Basic function of adding routes, view and user table

**Comment by caachz** [comment]
We decided as a team to move forward despite the failing travis ci. everything is passing locally.

File: app/controllers/welcome_controller.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class WelcomeController < ApplicationController
```
Comment (TravisBorgsmiller) [inline app/controllers/welcome_controller.rb:1]: Strong controller name

File: db/migrate/20200408221148_create_users.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+class CreateUsers < ActiveRecord::Migration[6.0]
```
Comment (TravisBorgsmiller) [inline db/migrate/20200408221148_create_users.rb:1]: This migration looks just like what we needed!

**Pull request merged**